### DISCUSSION TITLE
How to handle threat feeds

### DISCUSSION THREAD

**Discussion by wildlyunder** [Security-Onion-Solutions/securityonion#7456] [Unsupported Versions]
Hi,

We've got a few threat feeds and wondered how best we could use these with security onion. We use csv and JSON threat feeds which contain around 5k IOCs each

**Comment by weslambert** [comment]
Are you looking to search for IOCs retroactively, or going forward? Typically folks use threat feeds via the Zeek Intel Framework, Elastalert blacklist rules, of the Filebeat Threat Intel module (requires additional steps beyond just enabling, like using the enrich processor, etc -- I will be publishing something soon that covers this in detail)

**Reply by wildlyunder** [reply]
Both retroactively and going forward; the IOC's will include:

- FQDN
- IP's
- CIDR
- Domains
- Username
- Filenames
- File Paths
- URL Paths
- Hashs
